packages/ts-codegen/src/__tests__/context.test.ts

```diff
@@ -47,29 +118,88 @@ describe(generateTS, () => {
             }
           `,
           {
-            contextSubTypeNameTemplate: "I${resourceName}StateMachineContext",
-            contextSubTypePathTemplate:
-              "@msteams/core-cdl-sync-${resourceName}",
+            contextSubTypeMetadata: contextSubTypeMetadata,
+            defaultContextSubTypePath: "@package/default-context",
+            defaultContextSubTypeName: "DefaultContextType",
```

Comment:
given how it's used I think it's due to rename this to `baseContext*` or something with the same meaning

e.g. `baseContextTypePath` and `baseContextTypeName`